aggregators/aggregator_test.go

```diff
@@ -1187,11 +1188,8 @@ func BenchmarkAggregateCombinedMetrics(b *testing.B) {
 	}
 }
 
-func BenchmarkAggregateBatch(b *testing.B) {
-	logger, err := zap.NewDevelopment()
-	if err != nil {
-		b.Fatal(err)
-	}
+func benchmarkAggregateBatchConcurrent(b *testing.B, concurrency int) {
```

Comment:
Should we use https://pkg.go.dev/testing#B.RunParallel?

Reply:
Updated to have BenchmarkAggregateBatchSerial and BenchmarkAggregateBatchParallel